meta-sdk/dynamic-layers/qt5-layer/recipes-qt/qt5/qtbase_%.bbappend

```diff
@@ -39,8 +39,7 @@ QT_CONFIG_FLAGS_APPEND_imxpxp = "-no-eglfs"
 QT_CONFIG_FLAGS_APPEND_imxgpu2d = "-no-eglfs -no-opengl -linuxfb"
 QT_CONFIG_FLAGS_APPEND_imxgpu3d = "\
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11', '-no-eglfs', \
-        bb.utils.contains('DISTRO_FEATURES', 'wayland', '-no-eglfs', \
```

Comment:
@arulcse2008 if parsing doesn't restrict can't we do this in qt5-mel layer? that way relevant configs remain at one pace e.g
`QT_CONFIG_FLAGS_APPEND_imxgpu3d_remove = "-no-eglfs"`

Reply:
Yeah, I tried that change and that yields more compilation errors with QTFlags.